src/createReduxForm.js

```diff
@@ -170,115 +170,120 @@ type OnChangeFunction = (
   previousValues: Values
 ) => void
 
-export type Config = {
-  asyncBlurFields?: string[],
-  destroyOnUnmount?: boolean,
-  forceUnregisterOnUnmount?: boolean,
-  enableReinitialize?: boolean,
-  keepDirtyOnReinitialize?: boolean,
-  keepValues?: boolean,
-  form: string,
-  immutableProps?: string[],
-  initialValues?: Values,
-  getFormState?: GetFormState,
-  onChange?: OnChangeFunction,
-  onSubmit?: OnSubmitFunction,
-  onSubmitFail?: OnSubmitFail,
-  onSubmitSuccess?: OnSubmitSuccess,
-  propNamespace?: string,
-  submitAsSideEffect?: boolean,
-  validate?: ValidateFunction,
-  warn?: ValidateFunction,
-  touchOnBlur?: boolean,
-  touchOnChange?: boolean,
-  persistentSubmitErrors?: boolean,
-  registeredFields: any
```

Comment:
You forgot to put `registeredFields` back. It's missing in `Config`.

Reply:
It's in Props (in that big literal that comes after `OptionalConfig` in the intersection type chain). As far as I understand it isn't something that a caller passes into `@reduxForm`. Are you seeing any false positives from the typechecker?

Reply:
(Pushed up a new version with comments describing each type's purpose.)